packages/db/postgres/tables.sql

```diff
@@ -199,6 +199,22 @@ CREATE TABLE IF NOT EXISTS pin_sync_request
 
 CREATE INDEX IF NOT EXISTS pin_sync_request_pin_id_idx ON pin_sync_request (pin_id);
 
+-- Tracks pinning requests from Pinning Service API
+-- TODO(paolo) this table should be considered for track all content stored on web3.storage
+CREATE TABLE IF NOT EXISTS pa_pin_request
+(
+ id              BIGSERIAL PRIMARY KEY,
+ -- Points to the pinned content, it is updated once the content is actually being found.
+ content_cid     TEXT                                                         REFERENCES content (cid),
+  -- Points to auth key used to pin the content.
+ auth_key_id         BIGINT                                                       NOT NULL REFERENCES public.auth_key (id),
+ -- The id of the content being requested, it could not exist on IPFS (typo, node offline etc)
+ requested_cid   TEXT NOT NULL,
```

Comment:
For consistency with the `upload` table.

```suggestion
 source_cid   TEXT NOT NULL,
```

Reply:
Done in #810